gradle.properties

```diff
@@ -8,15 +8,15 @@ description=Support for litematica on client to worldedit on server communicatio
 modVersion=1.0.0
 
 # Fabric
-minecraftVersion=1.20.1
-loaderVersion=0.14.21
-yarnMappings=1.20.1+build.10
+minecraftVersion=1.20.4
+loaderVersion=0.15.11
+yarnMappings=1.20.4+build.3
 
 # Paper
 apiVersion=1.20
 
 # Dependencies
-fabricVersion=0.83.1+1.20.1
-paperVersion=1.20.1-R0.1-SNAPSHOT
+fabricVersion=0.97.1+1.20.4
+paperVersion=1.20.4-R0.1-SNAPSHOT
 litematicaFileId=4593296
```

Comment:
Update the litematica file id. Latest build is here:
https://www.curseforge.com/minecraft/mc-mods/litematica/files/5393557
So the new file id that we depend on is `5393557`

Reply:
However, shouldn't it be updated to the first Litematica version for 1.20.4?
https://www.curseforge.com/minecraft/mc-mods/litematica/files/4946471

Reply:
Our fabric.mod.json has dependency on any version of litematica. So this one is only to make sure that the litematica internals that we're using are still there. Generally, we would want users to use the latest.